scripts/buildPythonLibraries.sh

```diff
@@ -2,65 +2,86 @@
 
 # Utility script that will copy files over from your xml directory, then call
 # the salgenerator commands to build your SAL libraries. 
-XML_WORK_DIR="/gitdir/ts_xml" #Update to be the path to your xml repo.
+# optioanlly add  list or args to run after the Topic name e..g MTmount java lib 
+# validate and htmlo will be run any way -- defaul is python c++ lib as it was before this change.
+# Option -nc in FIRST position to not clean all the files from test
+ 
+if [ -z ${XML_WORK_DIR}x ]; then
```

Comment:
is the character "x" intended to be here? String will never be empty.

Reply:
yes if you have an empty variable the test will fail in a different way ..